sources/packages/backend/apps/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -265,4 +266,41 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       };
     });
   }
+
+  /**
+   * Gets all eligible application that can be requested for application
```

Comment:
Minor but the `application` should be plural to enforce the idea of a list being returned. Same for the service method.